cocos/renderer/CCGLProgramCache.cpp

```diff
@@ -483,6 +495,14 @@ void GLProgramCache::loadDefaultGLProgram(GLProgram *p, int type)
         case kShaderType_3DTerrain:
             p->initWithByteArrays(cc3D_Terrain_vert, cc3D_Terrain_frag);
             break;
+        case kShaderType_CameraClear:
+            p->initWithByteArrays(ccCameraClearVert, ccCameraClearFrag);
+            break;
+#if CC_TARGET_PLATFORM == CC_PLATFORM_WP8 || defined(WP8_SHADER_COMPILER)
```

Comment:
CC_PLATFORM_WP8 is not supported now, and corresponding macros are removed too.
So please don't use this macro.